test/hotspot/jtreg/runtime/exceptionMsgs/NoClassDefFoundError/libNoClassDefFoundErrorTest.c

```diff
@@ -42,3 +46,34 @@ Java_NoClassDefFoundErrorTest_callFindClass(JNIEnv *env, jclass klass, jstring c
 }
 
 
+static char* giant_string() {
+    size_t len = ((size_t)INT_MAX) + 3;
```

Comment:
I don't think we need to allocate and touch 2GB for this test. Lets just allocate 64K + x, that should hit the Symbol name limit as well.

Reply:
We already test that case, this is explicitly trying to test the INT_MAX issue where we would crash prior to this fix.